src/Fmt_ast.ml

```diff
@@ -1627,7 +1632,7 @@ and fmt_value_description c ctx vd =
         $ fmt " :@ " $ fmt_core_type c (sub_typ ~ctx pval_type)
         $ list_fl pval_prim (fun ~first ~last:_ s ->
               fmt_if first "@ =" $ fmt " \"" $ str s $ fmt "\"" ) )
-    $ fmt_attributes c (fmt "@;<2 2>") ~key:"@@" atrs (fmt "")
+    $ fmt_attributes c (fmt "@;<2 2>") ~box:false ~key:"@@" atrs (fmt "")
```

Comment:
Instead of removing the box, I suspect that `fmt "@;<2 2>"` should be evaluated before `fmt_attributes`:
```
$ fmt "@;<2 2>" $ fmt_attributes c (fmt "") ~key:"@@" atrs (fmt "")
```
(and I guess the `pre` arg of `fmt_attributes` ought to be optional)

Reply:
It causes issue with the docstring bellow. Do you mind giving it a try ?

Reply:
Sorry, you're right, what you had is better.